### DISCUSSION TITLE
Use Helmet type security in oak

### DISCUSSION THREAD

**Discussion by omymy1** [oakserver/oak#364] [Q&A]
How to use Use Helmet type security in oak

**Comment by omymy1** [comment]
Answer Me

**Reply by kitsonk** [reply]
Don't be rude.

**Comment by omymy1** [comment]
Sorry for short words because i am working, Please tell me security packages available for oak

**Reply by kitsonk** [reply]
I am not aware of any specifically. All Helmet does is make it easier to set security headers on a response. It just requires someone to create it.
